interpreter/cling/lib/Interpreter/IncrementalParser.h

```diff
@@ -102,10 +102,11 @@ namespace cling {
     typedef llvm::PointerIntPair<Transaction*, 2, EParseResult>
       ParseResultTransaction;
     IncrementalParser(Interpreter* interp, int argc, const char* const *argv,
-                      const char* llvmdir);
+                      const char* llvmdir, bool isChildInterpreter = false);
     ~IncrementalParser();
 
-    void Initialize(llvm::SmallVectorImpl<ParseResultTransaction>& result);
+    void Initialize(llvm::SmallVectorImpl<ParseResultTransaction>& result,
+                    bool childInterp = false);
```

Comment:
I don't think `childInterp` should have a default value, given that this function is only used internally, and called only once or twice in all of cling's code.